src/Symfony/Component/DependencyInjection/ChildDefinition.php

```diff
@@ -213,12 +213,14 @@ public function getArgument($index)
      */
     public function replaceArgument($index, $value)
```

Comment:
also ChildDefinition::getArgument doesn't work with string yet. whereas you changed the parent Definition::getArgument.